### PR TITLE
Update pattern terminal

### PR DESCRIPTION
**Pull request by slprime**
### 1. Change active area for ViewCellSlots
| before | after |
|--------|-------|
before - viewCell active area | after - viewCell active area

### 2. Change Scroll Area
| before | after |
|--------|-------|
before - change scroll area | after -change scroll area

### 3. Overlay
| before | after |
|--------|-------|
before - nei overlay viewCells | after - nei overlay viewCells

### 4. Switching Inverted Property
| before | after |
|--------|-------|
before - switch invert state | after - switch invert state

### 5. Clear Terminal before apply pattern
| before | after |
|--------|-------|
before - edit pattern | after - edit pattern

### 6. Autodetect pattern state
| before | after |
|--------|-------|
before - detect invert state | after- detect invert state

### 7. Edit Invalid Pattern
image

### 8. Add more slots to Processing Pattern Terminal (https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/10355)
image

P.S.
fix: https://github.com/GTNewHorizons/Applied-Energistics-2-Unofficial/issues/140

### CODE REVIEW

**Comment by slprime** [comment]
please don't merge. tomorrow i fix commit tree

**Comment by boubou19** [comment]
damn, the last one is insane. Well done!!!

**Comment by Connor-Colenso** [comment]
Amazing work.

File: src/main/java/appeng/client/gui/widgets/GuiScrollbar.java [hunk 1]
```diff
@@ -25,6 +25,10 @@
 
 public class GuiScrollbar implements IScrollSource
 {
+	private String txtBase = "minecraft";
+	private String txtFile = "gui/container/creative_inventory/tabs.png";
```
Comment (repo-alt) [inline src/main/java/appeng/client/gui/widgets/GuiScrollbar.java:29]: static final maybe

Comment (slprime) [reply src/main/java/appeng/client/gui/widgets/GuiScrollbar.java:29]: in [GuiPatternTermEx.java](https://github.com/GTNewHorizons/Applied-Energistics-2-Unofficial/pull/143/files#diff-432838dd0beada7a49e638133370b2c943c12de23f9cb63402ec3aa7a6db3a32) line 44 i change it for small scrollbar

**Comment by slprime** [comment]
@repo-alt no, there also only 2 pages, but now we can change it how many we need

**Review by repo-alt — APPROVED** [review]

**Comment by repo-alt** [comment]
(Indeed there are only 2 pages in 1.18. Don't think we actually need more right now anyway)

**Pull request merged**